tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,44 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe.only('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
+		givenLoggedInUser(before);
+		givenAnApplication(before);
+
+		before(async function () {
+			const apiKey = await balena.models.application.generateProvisioningKey(
+				this.application.id,
+			);
+			expect(apiKey).to.be.a('string').that.has.length(32);
+		});
+
+		// it('should fail when the application does not exist', function () {
+		// 	return expect(
+		// 		balena.models.apiKey.getProvisioningApiKeysByApplication(
+		// 			'nonExistentApp',
```

Comment:
```suggestion
		// 			'someorg/nonExistentApp',
```
Also, any reason this test case is commented out?